src/magick-image.ts

```diff
@@ -204,6 +207,16 @@ export class MagickImage extends NativeInstance implements IMagickImage {
             ImageMagick._api._MagickImage_BackgroundColor_Set(this._instance, valuePtr);
         });
     }
+    get borderColor(): MagickColor {
+        const colorPtr = ImageMagick._api._MagickImage_BorderColor_Get(this._instance);
+        return MagickColor._create(colorPtr);
+    }
+    set borderColor(value: MagickColor) {
+        value._use(valuePtr =>
+        {
```

Comment:
The `{` should be on the previous line to be consistent with the rest of the code.